### DISCUSSION TITLE
Change rustdesk server

### DISCUSSION THREAD

**Discussion by Andre0be** [rustdesk/rustdesk-server#303] [General]
Hello,

I have an old VPS server with rustdesk server 1.1.8 on it.

Now I have an new VPS server (1.1.8) and I want to change from old VPS to a new VPS.

I copy the public key (/opt/rustdesk/id_ed25519.pub) from the old to the new server and make the same with the private key (/opt/rustdesk/id_ed25519). Restart the server, but my clients say: connection error. this id does not exist.

But it is the right ID. What must I do, to change from an old server to an new one?

Thanks.

**Comment by dinger1986** [comment]
did you change the IP/DNS to the new server?

**Comment by Andre0be** [comment]
I use a subdomain and these I redirect. From old to the new server.

Rustdesk server is configured with dns name, from my subdomain.

I think the problem is the db_v2.sqlite3. In the old server are some entries. In the new server not.

**Reply by dinger1986** [reply]
It should just recreate it as long as they key matches.

Does the dns resolve correctly

**Comment by Andre0be** [comment]
yes. ping ip (ipv4 and ipv6) is right.

**Comment by dinger1986** [comment]
It's maybe trying to connect over ipv6

**Comment by Andre0be** [comment]
yes, but the ipv6 show the right server. ipv4 the same. 

is it enough to adjust the id_ed25519 or do the sqlite3 files also need to be copied (from the old server)?
